From 182230a4aca44731845317b5d28e2af2cc98a441 Mon Sep 17 00:00:00 2001
From: Georges Basile Stavracas Neto <georges.stavracas@gmail.com>
Date: Fri, 23 Feb 2024 20:04:12 -0300
Subject: [PATCH] hwdb: Add support for Elgato Stream Deck Plus

Add support for the following device:

 - ID 0fd9:0084 Elgato Systems GmbH Stream Deck Plus

(cherry picked from commit afdf63fcf2c6112278d6b0f456bfecca4d6ec3c7)

Related: RHEL-5950
---
 hwdb.d/70-av-production.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/70-av-production.hwdb b/hwdb.d/70-av-production.hwdb
index 9890d726e5..a701d1c1ea 100644
--- a/hwdb.d/70-av-production.hwdb
+++ b/hwdb.d/70-av-production.hwdb
@@ -56,6 +56,10 @@ usb:v0FD9p006D*
 usb:v0FD9p0080*
  ID_AV_PRODUCTION_CONTROLLER=1
 
+# Stream Deck Plus
+usb:v0FD9p0084*
+ ID_AV_PRODUCTION_CONTROLLER=1
+
 # Stream Deck Pedal
 usb:v0FD9p0086*
  ID_AV_PRODUCTION_CONTROLLER=1
